modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/ExecutionServiceImpl.java

```diff
@@ -311,11 +310,11 @@ private static RuntimeException convertDdlException(Throwable e) {
         }
 
         if (e instanceof IgniteInternalCheckedException) {
-            return new IgniteInternalException(DDL_EXEC_ERR, "Failed to execute DDL statement [stmt=" /*+ qry.sql()*/
+            return new IgniteInternalException(RUNTIME_EXECUTION_ERR, "Failed to execute DDL statement [stmt=" /*+ qry.sql()*/
                     + ", err=" + e.getMessage() + ']', e);
         }
 
-        return (e instanceof RuntimeException) ? (RuntimeException) e : new SqlException(DDL_EXEC_ERR, e);
+        return (e instanceof RuntimeException) ? (RuntimeException) e : new SqlException(RUNTIME_EXECUTION_ERR, e);
```

Comment:
Part of stacktrace could be lost if `e` was unwrapped from CompletionException few lines above `e = e.getCause();`
BTW, ExecutionException is never unwrapped. Seems, it is just missed.

Reply:
`CompletableFuture` wraps all exceptions in `CompletionException`, so we should not get `ExecutionException ` here, unless of course we have a custom `CompletableFuture/CompletionStage` that does bother itself such details.

Reply:
Ok, ExecutionException can be thrown from `future.get()` method, but we never intend calling synchronous methods in thread pools.